x/vesting/client/cli/tx.go

```diff
@@ -1,313 +1,209 @@
-// 		},
-// 	}
-
-// 	cmd.Flags().Bool(FlagMerge, false, "Merge new amount and schedule with existing ClawbackVestingAccount, if any")
-// 	cmd.Flags().String(FlagLockup, "", "path to file containing unlocking periods")
-// 	cmd.Flags().String(FlagVesting, "", "path to file containing vesting periods")
-// 	flags.AddTxFlagsToCmd(cmd)
-// 	return cmd
-// }
-
-// // NewMsgClawbackCmd returns a CLI command handler for creating a
-// // MsgClawback transaction.
-// func NewMsgClawbackCmd() *cobra.Command {
-// 	cmd := &cobra.Command{
-// 		Use:   "clawback [address]",
-// 		Short: "Transfer unvested amount out of a ClawbackVestingAccount.",
-// 		Long: `Must be requested by the original funder address (--from).
-// 		May provide a destination address (--dest), otherwise the coins return to the funder.
-// 		Delegated or undelegating staking tokens will be transferred in the delegated (undelegating) state.
-// 		The recipient is vulnerable to slashing, and must act to unbond the tokens if desired.`,
-// 		Args: cobra.ExactArgs(1),
-// 		RunE: func(cmd *cobra.Command, args []string) error {
-// 			clientCtx, err := client.GetClientTxContext(cmd)
-// 			if err != nil {
-// 				return err
-// 			}
-
-// 			addr, err := sdk.AccAddressFromBech32(args[0])
-// 			if err != nil {
-// 				return err
-// 			}
-
-// 			var dest sdk.AccAddress
-// 			destString, _ := cmd.Flags().GetString(FlagDest)
-// 			if destString != "" {
-// 				dest, err = sdk.AccAddressFromBech32(destString)
-// 				if err != nil {
-// 					return fmt.Errorf("bad dest address: %w", err)
-// 				}
-// 			}
-
-// 			msg := types.NewMsgClawback(clientCtx.GetFromAddress(), addr, dest)
-// 			if err := msg.ValidateBasic(); err != nil {
-// 				return err
-// 			}
-
-// 			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
-// 		},
-// 	}
-
-// 	cmd.Flags().String(FlagDest, "", "address of destination (defaults to funder)")
-// 	flags.AddTxFlagsToCmd(cmd)
-// 	return cmd
-// }
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkvesting "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+
+	"github.com/tharsis/evmos/x/vesting/types"
+)
+
+// Transaction command flags
+const (
+	FlagDelayed = "delayed"
+	FlagDest    = "dest"
+	FlagLockup  = "lockup"
+	FlagMerge   = "merge"
+	FlagVesting = "vesting"
+)
+
+// NewTxCmd returns a root CLI command handler for certain modules/vesting
+// transaction commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Vesting transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
```

Comment:
I think we need to add the txs from the SDK to create the other vesting account types

Reply:
I can add them, but I don't understand why we would need them. Aren't we specifically saying that we only want to work with ClawbackVestingAccounts?

Reply:
@fedekunzeI added `NewMsgCreateVestingAccountCmd` but I think we cannot add `NewCreatePeriodicVestingAccountCmd` because the sdk version that we are using (0.45.1) doesnt not include `sdkvesting.NewMsgCreatePeriodicVestingAccount`

Reply:
It's missing here: https://github.com/cosmos/cosmos-sdk/blob/v0.45.1/x/auth/vesting/types/msgs.go but they have it on their `master`